src/main/java/org/sonarsource/scanner/maven/bootstrap/ScannerBootstrapper.java

```diff
@@ -38,12 +40,16 @@
 import org.sonarsource.scanner.api.EmbeddedScanner;
 import org.sonarsource.scanner.api.ScanProperties;
 
+import static org.sonarsource.scanner.maven.bootstrap.MavenProjectConverter.getPropertyByKey;
+
 /**
  * Configure properties and bootstrap using SonarQube scanner API
  */
 public class ScannerBootstrapper {
 
   static final String UNSUPPORTED_BELOW_SONARQUBE_56_MESSAGE = "With SonarQube server prior to 5.6, use sonar-maven-plugin <= 3.3";
+  private static final String SONAR_HOST_URL_KEY = "sonar.host.url";
+  private static final String SONAR_HOST_URL = "https://sonarcloud.io";
```

Comment:
What about using SONARCLOUD in the name of the constant to be more explicit when reading the code where this constant is used?

```suggestion
  private static final String SONARCLOUD_HOST_URL = "https://sonarcloud.io";
```